### DISCUSSION TITLE
怎么样进行章节的交叉引用？

### DISCUSSION THREAD

**Discussion by WEIYujie1996** [sjtug/SJTUThesis#1051] [提问]
比如在chapter 1引用 chapter 3里面的section 2该怎么做？
比如常见的一个场景是，chapter 2是一章研究内容，需要交叉引用appendix里面的一个定理，如果app按照模板给的方式，是引用不了的？

**Comment by LogCreative** [comment]
直接 `\section{节}\label{sec:target}`，然后引用时 `第 \ref{sec:target} 节`/ `附录 \ref{sec:target}`。
